### DISCUSSION TITLE
无法编辑笔记中的表格内容

### DISCUSSION THREAD

**Discussion by lineex** [windingwind/zotero-better-notes#697] [Q&A] [answered]
使用zotero6，以下是笔记代码
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <style>
    table {
      width: 100%;
      border-collapse: collapse;
    }

    th, td {
      padding: 8px;
      border: 1px solid #ddd;
      white-space: nowrap;
      overflow: hidden;
      text-overflow: ellipsis;
    }
  </style>
</head>
<body>
  <h2>为什么</h2>

  <table>
    <tr>
      <th>文章题目</th>
      <th>研究目的</th>
      <th>研究设计</th>
      <th>参与者信息</th>
      <th>数据收集与分析</th>
      <th>结果与讨论</th>
      <th>结论</th>
      <th>引用文献</th>
    </tr>
    <tr>
      <td>文章题目1</td>
      <td>研究目的1</td>
      <td>研究设计1</td>
      <td>参与者信息1</td>
      <td>数据收集与分析1</td>
      <td>结果与讨论1</td>
      <td>结论1</td>
      <td>引用文献1</td>
    </tr>
    <tr>
      <td>文章题目2</td>
      <td>研究目的2</td>
      <td>研究设计2</td>
      <td>参与者信息2</td>
      <td>数据收集与分析2</td>
      <td>结果与讨论2</td>
      <td>结论2</td>
      <td>引用文献2</td>
    </tr>
  </table>

  <h2>是什么</h2>

  <table>
    <tr>
      <th>主题</th>
      <th>定义及时间</th>
      <th>临床意义</th>
      <th>关联文献</th>
    </tr>
    <tr>
      <td>主题1</td>
      <td>定义及时间1</td>
      <td>临床意义1</td>
      <td>关联文献1</td>
    </tr>
    <tr>
      <td>主题2</td>
      <td>定义及时间2</td>
      <td>临床意义2</td>
      <td>关联文献2</td>
    </tr>
  </table>

</body>
</html>

**Comment by windingwind** [comment, marked as answer]
善用搜索。z6 windows版本的bug，与插件无关
